templates/blast-form.html

```diff
@@ -80,8 +86,16 @@
       listen_for_fee_check();
     };
 
-    $('select[name="amount"]').change(function() {
+    $('input[name="amount"]').click(function() {
```

Comment:
Do we need this logic? If they're radio buttons, clicking one should automatically deselect the others. You should be able to use the `.change` listener for this.

In any case, if you keep this block, I'd recommend updating it to use some jQuery-specific stuff to ensure cross-browser compatibility:
- `$(this)` instead of `this`
- `$(this).attr(ATTR_NAME, ATTR_VALUE)` instead of `setAttribute/removeAttribute`
- `$(this).prop('checked, true/false)` instead of `this.checked`